src/components/SignalCard/card/creator/SignalCreator.js

```diff
@@ -22,7 +22,11 @@ const SignalCreator = ({
   const nameOrEmail = username || email
 
   return (
-    <Link to={'/profile/' + id} className={cx(styles.container, className)}>
+    <Link
+      to={'/profile/' + id}
+      className={cx(styles.container, className)}
+      onClick={e => e.stopPropagation()}
```

Comment:
better to move this function in onClick outside to avoid recreations

Reply:
agree